packages/kit/src/compute-limit-internal.ts

```diff
@@ -17,8 +17,8 @@ import { Blockhash, Commitment, Slot } from '@solana/rpc-types';
 import {
     appendTransactionMessageInstruction,
     CompilableTransactionMessage,
-    isDurableNonceTransaction,
```

Comment:
This bothered me too, but I didn't do it because it would require a bump to `@solana/kit` 3.0.

That said…

![Added via Giphy](https://media0.giphy.com/media/v1.Y2lkPWM5NDg3NzQzdDMzODFjZW1vaDh0amtiZ2p5dWIzbTJ3b2FveTdhYndwbHUxdTVmMCZlcD12MV9naWZzX3NlYXJjaCZjdD1n/wi8Ez1mwRcKGI/giphy.gif)